src/util/clevertap.js

```diff
@@ -484,6 +485,14 @@ export const handleEmailSubscription = (subscription, reEncoded, fetchGroups, ac
       url = addToURL(url, 'sub', subscription)
     }
 
-    request.fireRequest(url)
+    RequestDispatcher.fireRequest(url)
+  }
+}
+
+export const tcWrapper = function (f) {
```

Comment:
This will become circular dependency. Let's move it to some new utils file with no dependencies.

Reply:
updated